### DISCUSSION TITLE
How to Print 4MB Ordinals

### DISCUSSION THREAD

**Discussion by jbrace02** [ordinals/ord#1577] [Q&A] [answered]
Hello, 

I saw the tweet that we can inscribe 4MB files to block-chain. 

To do this, do we need to enter a new command when inscribing? Update the Ord wallet. 

I'm on windows btw. 

Any help is appreciated.

**Comment by raphjaph** [comment, marked as answer]
It's on the master branch of this repo so you would need to clone it and then build the binary with `cargo build --release`. The flag is `--no-limit` but you need to know that large transactions do not get relayed in the mempool. So you would need to find a miner willing to mine your transaction.
